jest.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   coverageThreshold: {
     global: {
       statements: 90,
-      branches: 90,
+      branches: 85,
```

Comment:
@steveodonovan and I did some work earlier today to bring our coverage back up to a safer level. Can you revert this please? (I've verified your PR is fine locally with this at 90). It's too easy to let this stuff slip otherwise.

Reply:
Great, yeah will do! Pained me to do this, thanks

Reply:
Right I've rebased and noticed 90+ percent coverage across the board now on average